.buildkite/release-pipelines/complete-code-freeze.yml

```diff
@@ -1,10 +1,26 @@
+common_params:
+  # Common plugin settings to use with the `plugins` key.
+  - &common_plugins
+    - automattic/a8c-ci-toolkit#2.18.1
+  # Common environment values to use with the `env` key.
+  - &common_env
+    # Be sure to also update the `.xcode-version` file when updating the Xcode image/version here
+    IMAGE_ID: xcode-15.1
+
 steps:
   - label: Complete Code Freeze
-    plugins:
-      - automattic/a8c-ci-toolkit#2.18.2
+    key: complete_code_freeze
+    plugins: *common_plugins
+    env: *common_env
     # The code freeze completion needs to run on macOS because it uses genstrings under the hood
     agents:
         queue: mac
-    env:
-      IMAGE_ID: xcode-15.1
     command: ".buildkite/commands/complete-code-freeze.sh $RELEASE_VERSION"
+
+  - label: Log Outdated Pods
+    depends_on: complete_code_freeze
+    plugins: *common_plugins
+    env: *common_env
+    agents:
+        queue: mac
```

Comment:
Can this step run on Linux?

Reply:
I haven't tried, but I think the practical answer is no.

This step uses CocoaPods. Being a Ruby gem, I think it should be possible to install it on Linux. It might even be possible for the `pod outdated` step to run, as I assume it only makes HTTP calls to the specs repos. But we need to run `pod install` in order to fetch Gutenberg, or `pod outdated` won't proceed (see [this build](https://buildkite.com/automattic/wordpress-ios/builds/20488#018dc4e0-c868-48d8-ba4f-476f2caea4fe))

![image](https://github.com/wordpress-mobile/WordPress-iOS/assets/1218433/47b21fa2-d7cb-4a38-b36e-9ff51ad0d5d7)

Reply:
The whole step in CI takes ~2m (see screenshot in the other comment above). So while it might be possible, even desirable, not to load the Mac queue with this check, the effect is minimal, also when considering it will run once every two weeks, or once a week if we move to weekly releases.